frontend/src/k8sTypes.ts

```diff
@@ -138,6 +138,7 @@ export type ServingRuntimeAnnotations = Partial<{
   'enable-route': string;
   'enable-auth': string;
   'modelmesh-enabled': 'true' | 'false';
+  'serving-runtime-scope': string;
```

Comment:
For new labels/annotations we introduce we should prefix them with `opendatahub.io/`